copter/source/docs/mission-command-list.rst

```diff
@@ -292,12 +292,13 @@ OR the change in heading (in degrees) when "rel/abs" field is "1"
 **Sec** - not supported.  meant to limit the rotation speed (in deg/sec)
 as the vehicle turns to the desired heading
 
-**Dir (1=CW, -1=CCW)** - Used only when relative heading is specified
+**Dir (1=CW, -1=CCW)** - Used when relative heading is specified
```

Comment:
I found this very difficult to read and a little confusing because of the word "Used" at the beginning I would suggest a slight change:

**Dir (1=CW, -1=CCW)** - When relative heading is specified
(i.e. "rel/abs" = "1") denotes whether the autopilot should add
(CW) or subtract (CCW) the degrees (Deg) from the current heading to
calculate the target heading. When absolute heading is specified
(i.e. "rel/abs" = "0") denotes CW/CCW direction. If this is set to "0",
the vehicle will always turn in direction that most quickly gets it
to the new target heading.

Maybe a clearer discription might be:

**Dir (1=CW, -1=CCW)** - Used to denote the direction of 
rotation to acheive the target angle. If this is set to "0",
the vehicle will always turn in direction that most quickly gets it
to the new target heading.

Reply:
Thank you.
I prefer simple one.